examples/mario/play_mario.html

```diff
@@ -38,9 +38,7 @@
     <script type="text/javascript" src="js/levels/11tunnel.js"></script>
     <script type="text/javascript" src="js/game.js"></script>
 
-    <py-script>
-from js import handTrack, setTimeout, requestAnimationFrame, player
-from pyodide import create_once_callable
+    <py-script style="display: none;">
```

Comment:
Why this change?

Reply:
Because sometimes the DOM renders the `pyscript`'s code as a part of the HTML text